elements/lisk-p2p/src/index.ts

```diff
@@ -12,6 +12,9 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-export * from './disconnect_status_codes';
-export * from './errors';
+export * from './constants';
+export * from './exceptions';
```

Comment:
do we need to export these for external users?

Reply:
Not for now I guess. @jondubois?

Reply:
Please check what exactly we were exposing and let's minimize the exporting value.
After being 1.0, removing the exported function/value will be breaking change and we want to avoid that